nilearn/plotting/find_cuts.py

```diff
@@ -196,6 +196,10 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     orig_data = np.abs(img.get_data())
     this_shape = orig_data.shape[axis]
 
+    if not isinstance(n_cuts, numbers.Number):
+        raise ValueError("The number of cuts should be a strictly positive integer. "
+                         "You provided n_cuts=%s " % n_cuts)
```

Comment:
Tiny comment it looks like the fully correct version is "greater **than** or equal **to** 1" and the alternative winning the google battle is "greater or equal **to** 1". See http://english.stackexchange.com/questions/38287/do-we-say-is-greater-or-equal-to-or-is-greater-or-equal-than for more details.